config/database.yml

```diff
@@ -29,35 +29,37 @@ development:
   # To create additional roles in postgres see `$ createuser --help`.
   # When left blank, postgres will use the default role. This is
   # the same name as the operating system user that initialized the database.
-  #username: wemeditate
+  username: postgres
```

Comment:
Is it necessary to change the database.yml file? If you revert this file are you still able to connect? And if not, then can you try removing the password from your postgres installation.